src/pages/LoginPage.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
```

Comment:
@Recmo I would like to have your opinion on this component. Yesterday we talked that maybe it makes sense to implement pages as one component without using any layouts. And later when we observe to much markup repetition we will create some generic layouts. This is outcome of first login page.